tensorboard/plugins/beholder/BUILD

```diff
@@ -8,43 +8,77 @@ licenses(["notice"])  # Apache 2.0
 exports_files(["LICENSE"])
 
 py_library(
-    name = "beholder_plugin",
-    srcs = [
-        "beholder_plugin.py",
-        "shared_config.py",
+    name = "beholder",
+    srcs = ["beholder.py"],
+    data = ["resources"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":file_system_tools",
+        ":im_util",
+        ":shared_config",
+        ":video_writing",
+        ":visualizer",
+        "//tensorboard:expect_numpy_installed",
+        "//tensorboard:expect_tensorflow_installed",
+        "//tensorboard/backend/event_processing:plugin_asset_util",
     ],
+)
+
+py_library(
+    name = "beholder_plugin",
+    srcs = ["beholder_plugin.py"],
     srcs_version = "PY2AND3",
     deps = [
         ":beholder",
         ":file_system_tools",
         ":im_util",
         ":shared_config",
+        "//tensorboard:expect_numpy_installed",
+        "//tensorboard:expect_tensorflow_installed",
         "//tensorboard/backend:http_util",
         "//tensorboard/backend/event_processing:plugin_asset_util",
         "//tensorboard/plugins:base_plugin",
         "@org_pocoo_werkzeug",
     ],
 )
 
+py_binary(
+    name = "beholder_demo",
+    srcs = ["beholder_demo.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":beholder",
+        "//tensorboard:expect_numpy_installed",
```

Comment:
another nit: The demo doesn't actually use numpy. It had previously just imported it. After #1045 is merged, we can remove that dep.

Reply:
Done, thanks!